composer/workflows/dataflow_template_operator_tutorial/create_dataset_and_table_test.py

```diff
@@ -0,0 +1,67 @@
+from google.cloud import bigquery
+import pytest
+import google.cloud.exceptions
+import uuid
+
+client = bigquery.Client()
+dataset_UUID = str(uuid.uuid4()).split("-")[0]
```

Comment:
Is it really needed to only use the first part of the uuid4? Can you just use the whole thing?

Reply:
I believe that bigquery doesn't like hyphens in dataset names which is why we're just using part here

Reply:
Correct, I had to fail a test to learn this the hard way:(